packages/xchain-radix/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@xchainjs/xchain-radix",
+  "version": "0.1.0",
+  "description": "Custom Radix client and utilities used by XChainJS clients",
+  "keywords": [
+    "XRD",
+    "Radix",
+    "XChain"
+  ],
+  "author": "RadixDLT",
+  "homepage": "https://github.com/xchainjs/xchainjs-lib",
+  "license": "MIT",
+  "main": "lib/index.js",
+  "module": "lib/index.esm.js",
+  "typings": "lib/index.d.ts",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git@github.com:xchainjs/xchainjs-lib.git"
+  },
+  "scripts": {
+    "clean": "rm -rf .turbo && rm -rf lib",
+    "build": "yarn clean && rollup -c",
```

Comment:
Add the "build:release" script the rest of the clients have

Reply:
done